### DISCUSSION TITLE
Auto zoom for navigation turns?

### DISCUSSION THREAD

**Discussion by habitoti** [osmandapp/OsmAnd#14175] [Ideas]
Hi,

just did my first (bike) trip with OSMAnd and a yearly map subscription, and it’s fantastic (though I still need to explore the gazillion of options…). 
Even on a bike it would be great if there would be an option to zoom automatically into an approaching turn (like most car navigation systems do). Sometimes these are tight winding small paths that are hard to see at bigger zoom levels (like winding right and going below a bridge, while it appears to be just a left turn at higher zoom). This should just slowly zoom in while getting closer to the actual turn, and then zoom back to level before once turn has been completed.
Or is that even already available and I just couldn‘t find within the tons of settings?

Thanks, habitoti

**Comment by vshcherb** [comment]
Auto zoom now is dependent only on speed and not on road information

**Reply by habitoti** [reply]
I know, but that‘t doesn‘t make much sense for cycling. That‘s why I suggest this option to tie it to approaching the next turn information during navigation (as also announced via the turn arrow „widget“). So like when approx 100m away from next turn, it could start zooming in (and right back to previous level when turn was completed).

**Reply by vshcherb** [reply]
Sounds interesting actually, I like your suggestion.
